src/NHSD.BuyingCatalogue.Organisations.Api/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using HealthChecks.Network.Core;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using NHSD.BuyingCatalogue.Identity.Common.Constants;
+using NHSD.BuyingCatalogue.Organisations.Api.Settings;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.Extensions
+{
+    internal static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection RegisterHealthChecks(this IServiceCollection services, string connectionString, SmtpSettings smtpSettings)
+        {
+            if (connectionString is null) throw new ArgumentNullException(nameof(connectionString));
+
+            if (smtpSettings is null) throw new ArgumentNullException(nameof(smtpSettings));
```

Comment:
Can the `throw` statements be moved to a new line? E.g.

```c#
if (connectionString is null)
    throw new ArgumentNullException(nameof(connectionString));
```
I think this is improves the readability.

Reply:
👍 I don't have a problem moving them to a new line.